app/decorators/content_items_decorator.rb

```diff
@@ -2,11 +2,24 @@ class ContentItemsDecorator < Draper::CollectionDecorator
   delegate :current_page, :total_pages, :limit_value, :entry_name, :total_count, :offset_value, :last_page?
 
   def header
-    slug = helpers.params[:organisation_slug]
-    if slug
+    if slug.present? && taxonomy.present?
+      "#{Organisation.find_by(slug: slug).title} + #{taxonomy}"
+    elsif slug.present?
       Organisation.find_by(slug: slug).title
+    elsif taxonomy.present?
+      taxonomy
     else
-      'GOV.UK'
+      "GOV.UK"
     end
   end
+
+private
+
+  def slug
+    helpers.params[:organisation_slug]
+  end
+
+  def taxonomy
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/95/commits/e5ede1820800f155f254f537dc4d17e94842d30d